joomla/joomla-cms
# [5.1] Convert mod_latestactions to service provider

### Summary of Changes

Converts the administrator latest actions module to service provider.

### Testing Instructions

1. Make sure you have administrator module mod_latestactions published
2. You should see module output in administrator area
3. You should not see module output, when logged in to administrator without SuperUser permissions

![mod_latestactions](https://github.com/joomla/joomla-cms/assets/59166858/e6e3f8f7-5c6c-4eb9-ab8d-e545b4ffa158)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
